packages/web-components/src/spinner/spinner.template.ts

```diff
@@ -2,9 +2,17 @@ import { html } from '@microsoft/fast-element';
 import { Spinner } from './spinner.js';
 
 export const template = html<Spinner>`
-  <slot name="indicator"
-    ><svg class="progress" part="progress" viewBox="0 0 16 16">
-      <circle class="background" cx="8px" cy="8px" r="7px"></circle>
-      <circle class="indicator" cx="8px" cy="8px" r="7px"></circle></svg
-  ></slot>
+  <slot name="indicator">
+    <div class="background"></div>
```

Comment:
Should we have `part` on any of these?

Reply:
This is a good question! My initial thinking is that named `part` hooks should be more permanent and not inside default slotted content which could be overwritten. But my opinion there isn't very strong. Happy to chat about it more next week.

Reply:
Resolving this per discussion. Will open another issue around evolving a general API design philosophy on parts.

per @chrisdholt: 
> I think we should move ahead with the knowledge that ideally the semantic tokens resolves this...or that parts can be added later. We don't have a clear story for what we want to do there technically. 

We can develop that story and/or wait until this feature is requested.